config/environments/test.rb

```diff
@@ -18,7 +18,7 @@
   # Configure public file server for tests with Cache-Control for performance.
   config.public_file_server.enabled = true
   config.public_file_server.headers = {
-    "Cache-Control" => "public, max-age=#{1.hour.to_i}"
+    "Cache-Control" => "public, max-age=3600"
```

Comment:
What's the reason for this change?

Reply:
It seems to execute this before the active support helpers are loaded allowing this to work. It didn't seem like a big deal to hard-code it.

Reply:
Interesting.  The default hasn't changed, so I'm not sure why it wouldn't work, but also not a big deal.  :)

Reply:
I was able to revert this because of https://github.com/CruGlobal/text-steps-widget/pull/6/files#diff-77f322f5ab0c54b1f2793b339574333dc55889645d910a93ede0fd7aa13b217aR3